### ISSUE TITLE
Need for a column called "probe" in "annot" for "molecular.subtyping" and "intrinsic.cluster.predict" but not "oncotypedx" and "gene70"

### ISSUE THREAD

**Issue by PedroMilanezAlmeida** [bhklab/genefu#15]
Hey y'all,

It would be great if the help of "molecular.subtyping" and "intrinsic.cluster.predict" would indicate the need for a column called "probe" in "annot".

Right now it says only that at least one column named "EntrezGene.ID" (for ssp, scm, AIMS, and claudinLow models) or "Gene.Symbol" (for the intClust model) is needed, but a column called "probe" is also needed.

This is problematic because other functions such as "oncotypedx" and "gene70" require the rownames of "annot" to be the probe names, instead of a column called "probe" with the probe names.

It would be even better if this was unified (e.g., either "annot" always needs a "probe" column or it never does).

Cheers and thanks!

**Comment by Puriney** [comment]
Yes. The first column of the `annot` should be "probe", otherwise error shows up:
>Error in dimnames(x) <- dn : 
  length of 'dimnames' [1] not equal to array extent

**Comment by ChristopherEeles** [comment]
Hi @Puriney,

I will address this issue in the next update. It will likely take a few weeks though, as I am currently occupied with other projects.

Best,
Christopher Eeles
Software Developer
Benjamin Haibe-Kains Lab
Princess Margaret Cancer Centre

**Comment by Puriney** [comment]
No worry. The quick solution exists. We end-users can simply add one column named 'probe' and then the problem is solved. This issue, I guess, is to simply remind you to correct the description in the document.
